data/data/powervs/cluster/bootstrap/lb/main.tf

```diff
@@ -0,0 +1,53 @@
+# Using explicit depends_on as otherwise there are issues with updating and adding of pool members
```

Comment:
The bootstrap node, here, is part of the cluster stage. Looking at https://github.com/openshift/installer/blob/master/pkg/terraform/stages/powervs/stages.go#L10-L16 I don't see any bootstrap destroy functionality.

Is this intentional? Typically the bootstrap node and all resources that should be destroyed along with it will go into their own stage. Nothing else should go into that stage as it would be destroyed.

Reply:
oh actually, now that i'm not in a meeting and multitasking -- yes, this specific bit was intentional, as in, i didn't add a bootstrap phase and add these resources (vpc, network, cloud connection) all in the same PR. so i did put these into their own file now to go ahead and organize them in a way that he didn't have to do a major rebase.